lib/cylc/suite_srv_files_mgr.py

```diff
@@ -385,6 +385,8 @@ def parse_suite_arg(self, options, arg):
         If arg is a file, suite name is the base name of its container
         directory.
         """
+        if arg == '.':
+            arg = os.path.realpath(os.getcwd())
```

Comment:
Is `os.path.realpath` an overkill here?

Reply:
Quite possibly, would you prefer it without?

Reply:
Best without, if possible.